[2024-08-21T12:53:56.397+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-21T12:53:56.417+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 3_models_dag.prepare_data manual__2024-08-21T07:53:46.188493+00:00 [queued]>
[2024-08-21T12:53:56.424+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 3_models_dag.prepare_data manual__2024-08-21T07:53:46.188493+00:00 [queued]>
[2024-08-21T12:53:56.424+0500] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-21T12:53:56.435+0500] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): prepare_data> on 2024-08-21 07:53:46.188493+00:00
[2024-08-21T12:53:56.446+0500] {logging_mixin.py:190} WARNING - /home/nicolaspc/anaconda3/envs/mlops_env/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=7727) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-21T12:53:56.447+0500] {standard_task_runner.py:72} INFO - Started process 7756 to run task
[2024-08-21T12:53:56.447+0500] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '3_models_dag', 'prepare_data', 'manual__2024-08-21T07:53:46.188493+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/.ipynb_checkpoints/train_models-2-checkpoint.py', '--cfg-path', '/tmp/tmpiy6fopea']
[2024-08-21T12:53:56.448+0500] {standard_task_runner.py:105} INFO - Job 92: Subtask prepare_data
[2024-08-21T12:53:56.500+0500] {task_command.py:467} INFO - Running <TaskInstance: 3_models_dag.prepare_data manual__2024-08-21T07:53:46.188493+00:00 [running]> on host nicolaspc-HP-Pavilion-Laptop-15-eh1xxx
[2024-08-21T12:53:56.580+0500] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nicolas Pal' AIRFLOW_CTX_DAG_ID='3_models_dag' AIRFLOW_CTX_TASK_ID='prepare_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-21T07:53:46.188493+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-21T07:53:46.188493+00:00'
[2024-08-21T12:53:56.581+0500] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-21T12:53:56.605+0500] {s3.py:1387} INFO - Downloading source S3 file from Bucket test-bucket-nicolas-1 with path datasets/california_housing.pkl
[2024-08-21T12:53:56.618+0500] {base.py:84} INFO - Retrieving connection 'aws_default'
[2024-08-21T12:53:56.619+0500] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-08-21T12:54:00.660+0500] {train_models-2-checkpoint.py:120} INFO - Data preparation finished.
[2024-08-21T12:54:00.664+0500] {python.py:240} INFO - Done. Returned value was: {'start_tiemstamp': '20240821 12:53', 'data_download_start': '20240821 12:53', 'data_download_end': '20240821 12:53', 'data_preparation_end': '20240821 12:54'}
[2024-08-21T12:54:00.701+0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-21T12:54:00.701+0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=3_models_dag, task_id=prepare_data, run_id=manual__2024-08-21T07:53:46.188493+00:00, execution_date=20240821T075346, start_date=20240821T075356, end_date=20240821T075400
[2024-08-21T12:54:00.756+0500] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-21T12:54:00.783+0500] {taskinstance.py:3891} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2024-08-21T12:54:00.786+0500] {local_task_job_runner.py:240} INFO - ::endgroup::
